vertx-web/src/main/java/io/vertx/ext/web/handler/sse/SSEPacket.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.vertx.ext.web.handler.sse;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.handler.sse.SSEHeaders;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+class SSEPacket {
```

Comment:
This class should be combined with the `SSEHeaders` so we have a single `DataObject` to describe the Event.